### PR TITLE
tools/lightning: update wording for multiple lighting docs

### PR DESCRIPTION
**Pull request by dcalvin**
Follow-up editing on https://github.com/pingcap/docs/pull/1087.
Involve multiple edits on formatting and wording for consistency and accuracy.

### CODE REVIEW

File: tools/lightning/deployment.md [hunk 1]
```diff
@@ -238,10 +238,10 @@ Download the TiDB-Lightning package (choose the same version as that of the TiDB
     # Maximum number of open engines.
     max-open-engines = 8
     # Maximum upload speed (bytes per second) from Importer to TiKV.
-    #upload-speed-limit = "512MB"
-    # minimum ratio of target store available space: store_available_space / store_capacity.
+    # upload-speed-limit = "512MB"
+    # Minimum ratio of available space on the target store: store_available_space / store_capacity.
```
Comment (CaitinChen) [inline tools/lightning/deployment.md]:
```suggestion
    # Minimum ratio of available space on the target store: `store_available_space`/`store_capacity`.
```

File: tools/lightning/deployment.md [hunk 2]
```diff
@@ -238,10 +238,10 @@ Download the TiDB-Lightning package (choose the same version as that of the TiDB
...
     # Importer pauses uploading SST if the availability ratio of the target store is less than this
-    # value, to give PD enough time to balance regions.
+    # value, to allow enough time for PD to balance regions.
```
Comment (CaitinChen) [inline tools/lightning/deployment.md]:
```suggestion
    # value, to allow enough time for PD to balance Regions.
```

**Comment by dcalvin** [comment]
PTAL again @CaitinChen Thanks.

**Comment by morgo** [comment]
@dcalvin There is a merge conflict, please resolve

**Comment by CaitinChen** [comment]
Please help remove the spaces before and after "/". Thanks.

**Pull request merged**